Resources/Textures/Constructible/Structures/Doors/hidden_door.rsi/meta.json

```diff
@@ -0,0 +1,149 @@
+{
+  "version": 1,
```

Comment:
is the hidden door-stuff intended to be in this PR?

Reply:
I was going to include it in this PR and started the work but didn't finish...so still on the fence on finishing it off but the resources are there. Will get round to it.

Reply:
Removed.